app/plugins/views.js

```diff
@@ -16,14 +16,23 @@ module.exports = {
           }
         },
         prepare: (options, next) => {
-          options.compileOptions.environment = nunjucks.configure([
+          const nunjucksAppEnv = nunjucks.configure([
             path.join(options.relativeTo || process.cwd(), options.path),
-            'node_modules/govuk-frontend/'
+            'node_modules/govuk-frontend/',
+            'node_modules/@ministryofjustice/frontend/'
           ], {
             autoescape: true,
             watch: false
           })
 
+          // Add filters from MOJ Frontend
```

Comment:
Could this be separated out into its own module?

Reply:
sure.